requirements-dev.txt

```diff
@@ -1,4 +1,3 @@
-black_nbconvert
```

Comment:
Seems like you removed `black_nbconvert`? It's needed for [NB styling](https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style)

Reply:
Now that Travis cache issue is resolved, couldn't you add back _black_nbconvert_ to the file?